nss-day-cohort-45/hello-world-column-counts
# added images/canada.png; added canada city, landmark, and citizen lists

## changes:
1) added canada.png to images/ directory
2) populated unordered lists of cities, landmarks, and citizens on canada.html with links to external sites for each

## testing:
1) serve branch and open canada.html in browser
2) verify that canada.png appears above canada headline
3) verify that lists populate with links to external sites for each list item
